schema/grype/db/blob/json/schema-latest.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "anchore.io/schema/grype/db/blob/json/6.1.1",
+  "$id": "anchore.io/schema/grype/db/blob/json/6.1.2",
```

Comment:
This might be an unintended consequence of the way we're doing schema numbering. The blobs didn't change, but we added a table, so we have a no-op bump in this number. Should we change it while it's still early days on publishing this schema?